package.json

```diff
@@ -64,7 +64,7 @@
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.2.1 || ^5.0.0",
-    "typescript": "^3.9"
+    "typescript": "^3.9 || ^4.0.0"
```

Comment:
What's going on here exactly? Renovate is smart enough to figure this out? This is a regression bug fix.

Reply:
Before https://github.com/standard/eslint-config-standard-with-typescript/issues/596
We supported `typescript>=3.9.0` which means that we also supported `typescript@4.0.0` but after your PR, you did a regression and we didn't supported anymore `typescript@4.0.0`.

Thanks to this PR, we support again `typescript@4.0.0`.

Reply:
Exactly. This is why #604 .